### DISCUSSION TITLE
Scrapli CLI Runner

### DISCUSSION THREAD

**Discussion by tybruno** [carlmontanari/scrapli#126] [Q&A] [answered]
Hello,

I absolutely love Scrapli! The code design is far superior than other similar libraries.

Is there a Scrapli CLI runner in development that networking engineers without programming knowledge can use to run commands on multiple devices? 

Use Case: 
As a networking engineer, I would like to run a list of commands on a list of devices via CLI terminal, so that I can get the running-config or configure multiple devices without needing to login to each device.

**Comment by carlmontanari** [comment, marked as answer]
Hey @tybruno! Thanks for the kind words!

You have excellent timing :) @hellt has recently created [commando](https://github.com/hellt/cmdo) which does (I think/hope) exactly what you are looking for! It is built on the go version of scrapli. Being that its go, this should be pretty easily used by anyone without having to install any stuff via pip/fuss with virtual environments and stuff like that. Roman's docs are, as always excellent, and should get ya easily setup to test it out.

Let us know if that works for the use case you are describing?

Carl

**Reply by tybruno** [reply]
Awesome! Thank you so much :) I will take a look and get back to you.
